pkg/cryptutil/kek.go

```diff
@@ -4,10 +4,9 @@ import (
 	"crypto/rand"
 	"fmt"
 
+	"github.com/akamensky/base58"
```

Comment:
So for some context, we removed references to the the original base58 library (and directly embedded it) because people didn't like our dependency on a bitcoin library. Assuming this library produces the same results I suppose that's better.

Reply:
I probably should have mentioned it in the description - I swapped this library out because profiling revealed the base58 encoding (primarily bigint math) as a significant portion of the runtime for this, so I found this library which has some optimizations that result in big performance gains

Reply:
That said, it's still super slow, so perhaps it might be a good idea to switch to a different encoding scheme in the future

Reply:
`base58` was used as a dense encoding that's file safe. Certainly open to alternatives.

Reply:
I think [base62](https://pkg.go.dev/github.com/jxskiss/base62) might be a good option. We can't swap out all usage of base58 but I think we could this one if we want to.

Reply:
Might be worth benchmarking :eyes: